<?xml version="1.0"?>
<launch>
  <node name="registration_server" pkg="halcon_package" type="registration_server.py" output="screen">  </node>
  <!-- <node name="registration_client" pkg="halcon_package" type="registration_client.py" output="screen">  </node> -->
</launch>
